lonestar/analytics/cpu/sssp/SSSP.cpp

```diff
@@ -80,8 +82,9 @@ static cll::opt<Algo>
                      clEnumVal(serDelta, "serDelta"),
                      clEnumVal(dijkstraTile, "dijkstraTile"),
                      clEnumVal(dijkstra, "dijkstra"), clEnumVal(topo, "topo"),
-                     clEnumVal(topoTile, "topoTile")),
-         cll::init(deltaTile));
+                     clEnumVal(topoTile, "topoTile"),
+                     clEnumVal(AutoAlgo, "choose between the algorithms automatically")),
```

Comment:
nitpick: "among" algorithms, not between

Reply:
Good point